src/Views/Components/ListTable/ListTablePage/index.tsx

```diff
@@ -82,13 +78,21 @@ export default function ListTablePage({
     const [selectedNames, setSelectedNames] = useState([]);
     const dialog = useRef() as {current: A11yDialogInstance};
     const checkboxRefs = useRef([]);
+    const [sortField, setSortField] = useState<{sortColumn: string; sortDirection: string}>({
+        sortColumn: 'id',
+        sortDirection: 'desc',
+    });
+    const {sortColumn, sortDirection} = sortField;
+    const locale = navigator.language || (navigator.languages || ['en'])[0];
```

Comment:
If you don't have the user's locale via browser, leave it for the backend to handle defaults instead of hardcoding it to `en`. We'll be using the WP locale as a fallback; even if it fails, we'll have the server locale.

Reply:
Resolved: https://github.com/impress-org/givewp/commit/c7bd39899e3bb0ad6404bcf731604d865012c176